2021-05-28 22:15:49 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-28 23:16:01 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-29 22:13:18 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-29 22:13:24 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-05-29 22:13:39 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - {'ret': 0, 'msg': '用户test的禁用状态更新成功'} , 操作者：admin

2021-05-29 22:13:52 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - {'ret': 0, 'msg': '用户test的禁用状态更新成功'} , 操作者：admin

2021-05-29 22:13:57 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 禁用状态更新失败，无法禁用超级用户
2021-05-29 22:13:57 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - {'ret': 1, 'msg': '禁用状态更新失败，无法禁用超级用户'} , 操作者：admin

2021-05-29 22:14:07 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - {'ret': 0, 'msg': '用户test的禁用状态更新成功'} , 操作者：admin

2021-05-29 22:14:41 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：test

2021-05-29 22:14:48 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：test

2021-05-29 22:15:11 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：test

2021-05-29 22:16:28 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-05-29 22:16:40 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户密码修改失败，跟原密码匹配失败 , 操作者：admin

2021-05-29 22:16:56 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：test

2021-05-29 22:17:07 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：test

2021-05-29 22:23:22 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-29 22:23:42 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户test的密码已更新 , 操作者：admin

2021-05-29 22:23:52 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 非管理员登陆，操作者为：test

2021-05-29 22:24:14 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：test

2021-05-29 22:24:39 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户test的密码已更新 , 操作者：admin

2021-05-29 22:28:23 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-29 22:28:26 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - {'ret': 1, 'msg': '用户密码修改失败，跟原密码匹配失败'} , 操作者：admin

2021-05-29 22:28:37 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：test

2021-05-29 22:28:41 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：test

2021-05-29 22:46:05 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-29 22:46:10 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：test

2021-05-29 22:46:25 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-05-29 22:48:10 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-29 22:48:14 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 禁用状态更新失败，无法禁用超级用户 , 操作者：admin

2021-05-30 22:45:20 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 22:45:24 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 该路由发生异常:'bool' object has no attribute 'password_md5'

2021-05-30 22:46:22 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 22:47:11 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:47:14 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:48:09 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:48:39 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 22:48:40 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:48:41 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:49:14 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 22:49:16 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:50:15 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 22:50:16 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 22:50:33 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 22:50:35 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-05-30 23:03:09 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 23:03:14 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-05-30 23:03:22 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 该路由发生异常:Incorrect padding

2021-05-30 23:03:27 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 该路由发生异常:Invalid base64-encoded string: number of data characters (5) cannot be 1 more than a multiple of 4

2021-05-30 23:07:24 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 23:07:27 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 该路由发生异常:Invalid base64-encoded string: number of data characters (5) cannot be 1 more than a multiple of 4

2021-05-30 23:08:30 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 23:08:35 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 该路由发生异常:Invalid base64-encoded string: number of data characters (5) cannot be 1 more than a multiple of 4

2021-05-30 23:09:08 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 23:09:23 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 23:09:25 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 该路由发生异常:Invalid base64-encoded string: number of data characters (5) cannot be 1 more than a multiple of 4

2021-05-30 23:10:27 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-05-30 23:10:30 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-01 19:58:06 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-01 19:58:10 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 登陆失败，用户名或者密码错误，操作者为：admin

2021-06-01 19:58:58 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-01 19:59:45 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 指定用户参数获取成功 , 操作者：admin

2021-06-01 20:01:15 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 指定用户参数获取成功 , 操作者：admin

2021-06-01 20:02:30 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 指定用户参数获取成功 , 操作者：admin

2021-06-01 22:52:51 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-01 22:52:52 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-01 22:53:00 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户密码修改失败，跟原密码匹配失败 , 操作者：admin

2021-06-01 22:54:38 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户test的密码已更新 , 操作者：admin

2021-06-06 21:23:45 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-06 21:24:11 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 模板删除失败，发生异常：mouldlist matching query does not exist.

2021-06-06 21:24:30 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test 已添加成功

2021-06-06 21:30:06 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:31:03 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:32:37 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:33:04 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:33:17 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:33:24 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:35:16 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:35:34 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:36:21 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:36:25 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:37:06 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:37:34 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:38:19 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-06 21:38:24 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test 已添加成功

2021-06-07 18:15:39 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:15:44 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:15:47 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:35:38 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:38:43 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:41:16 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:41:31 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:42:13 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:42:18 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:51:35 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:58:07 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 18:59:19 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-07 19:00:29 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test 已添加成功

2021-06-07 19:01:28 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test1 已添加成功

2021-06-07 19:01:31 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test11 已添加成功

2021-06-07 19:01:33 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test111 已添加成功

2021-06-07 19:01:35 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test1111 已添加成功

2021-06-07 19:01:38 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 模板名称:test11111 已添加成功

2021-06-07 19:04:54 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:08:40 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:08:41 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-07 19:10:24 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:12:10 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:12:57 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:12:58 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:argument of type 'Request' is not iterable

2021-06-07 19:14:18 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:14:19 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'Request' object is not subscriptable

2021-06-07 19:15:06 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:15:08 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:Related Field got invalid lookup: contains

2021-06-07 19:21:42 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:21:45 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:Related Field got invalid lookup: contains

2021-06-07 19:22:31 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:22:32 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'Request' object is not subscriptable

2021-06-07 19:26:31 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:26:56 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:Related Field got invalid lookup: contains

2021-06-07 19:27:29 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:27:30 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'Request' object is not subscriptable

2021-06-07 19:27:59 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:28:16 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:Related Field got invalid lookup: contains

2021-06-07 19:32:10 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:35:21 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-07 19:38:54 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-07 19:40:02 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 执行用户添加操作，操作者为：test
2021-06-07 19:40:02 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 用户添加失败,登录名 test 已经存在

2021-06-07 19:40:23 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 执行用户添加操作，操作者为：test1

2021-06-07 19:40:23 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户test1添加成功,即时生效

2021-06-07 19:41:21 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 执行用户删除操作，操作者为：admin

2021-06-07 19:44:08 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 模板删除失败，发生异常：mouldlist matching query does not exist.

2021-06-07 19:50:00 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 指定用户参数获取成功 , 操作者：admin

2021-06-07 19:55:01 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户密码修改失败，跟原密码匹配失败 , 操作者：admin

2021-06-07 19:55:09 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户密码修改失败，跟原密码匹配失败 , 操作者：admin

2021-06-07 19:55:22 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户密码修改失败，跟原密码匹配失败 , 操作者：admin

2021-06-07 19:55:57 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 用户test的密码已更新 , 操作者：admin

2021-06-07 19:57:23 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：test

2021-06-07 19:57:32 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-10 16:02:12 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-10 16:02:18 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-10 16:02:37 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-10 16:03:26 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-10 16:03:56 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-10 16:05:05 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-10 16:06:22 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-10 16:06:34 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:local variable 'mname' referenced before assignment

2021-06-10 16:07:17 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-10 16:10:04 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:39:23 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-15 14:39:33 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:42:01 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:42:03 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:43:15 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:43:17 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:50:11 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:51:36 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:51:38 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:51:41 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:51:46 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:52:40 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:52:43 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:54:57 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:54:58 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:11" - error - ERROR - 指定模板参数获取失败，发生异常:'User' object has no attribute 'mouldname'

2021-06-15 14:55:38 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:55:46 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-06-15 14:56:05 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:56:53 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:57:30 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-06-15 14:59:48 - logs - "/mnt/hgfs/WORKSMB/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-02 16:35:52 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-02 16:35:59 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-07-02 16:48:53 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-02 17:05:06 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-02 17:05:11 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-07-23 11:57:11 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 11:57:22 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 11:57:27 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 11:58:21 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 11:59:05 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 12:02:45 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 12:03:23 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-07-23 12:04:03 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-23 12:05:51 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:29:31 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:31:39 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:36:39 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:38:10 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:40:17 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:41:58 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:42:27 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:47:24 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:47:48 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-28 19:48:07 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 20:49:36 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 20:51:29 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 20:54:35 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:00:43 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:01:30 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:05:55 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:13:28 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:15:36 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:32:31 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-29 21:32:55 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:12:28 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:18:02 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:19:04 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-07-31 19:19:08 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin
2021-07-31 19:19:09 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 管理员登陆成功，操作者为：admin

2021-07-31 19:20:12 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:21:31 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:21:52 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:23:07 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:24:18 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:41:26 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:42:59 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:44:30 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:45:15 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:49:08 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:50:14 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:50:45 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:51:44 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 19:56:55 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 20:24:30 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

2021-07-31 21:10:18 - logs - "/home/awen/work/zjsys/adminapi/logdb.py:13" - debug - DEBUG - 加载路由表...

